lib/tasks/claims.rake

```diff
@@ -8,11 +8,14 @@ namespace :claims do
   desc "Create demo claim data for all states, allocating to case work as required"
   task :demo_data => :environment do
 
+    ADVOCATE_COUNT = 4
+    CLAIMS_PER_ADVOCATE_PER_STATE = 3
+
     example_advocate = Advocate.find(1)
```

Comment:
sure, i will work on it. was thinking of adding error handling

On 14 May 2015 at 16:31, Stephen Strudwick <notifications@github.com> wrote:

> In lib/tasks/claims.rake
> <https://github.com/ministryofjustice/crime-billing-online/pull/70#discussion_r30330803>
> :
>
> > @@ -8,11 +8,14 @@ namespace :claims do
> >    desc "Create demo claim data for all states, allocating to case work as required"
> >    task :demo_data => :environment do
> >
> > +    ADVOCATE_COUNT = 4
> > +    CLAIMS_PER_ADVOCATE_PER_STATE = 3
> > +
> >      example_advocate = Advocate.find(1)
>
> should we check for failure ? if no user find fail gracefully or do
> something useful like create one ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/crime-billing-online/pull/70/files#r30330803>
> .
>